x-pack/test/plugin_api_perf/plugins/task_manager_performance/server/init_routes.ts

```diff
@@ -75,9 +75,9 @@ export function initRoutes(
         }, Promise.resolve<ConcreteTaskInstance[] | undefined>(undefined));
 
       return res.ok({
-        body: await new Promise((resolve) => {
+        body: await new Promise<PerfResult>((resolve) => {
           setTimeout(() => {
-            performanceApi.endCapture().then((perf: PerfResult) => resolve(perf));
+            performanceApi.endCapture().then((perf) => resolve(perf));
```

Comment:
Did the change slip in or is it intentionally part of this work?

Reply:
Nope, moved the type annotation to `new Promise<PerfResult>`. This was done to narrow type inference and satisfy TS.